src/utils/config_filters.hpp

```diff
@@ -67,6 +67,10 @@ bool int_matches_if_present_or_negative(
 bool string_matches_if_present(
 	const config& filter, const config& cfg, const std::string& attribute, const std::string& def);
 
+bool list_string_matches_if_present(const config& filter, const config& cfg, const std::string& attribute);
```

Comment:
Draft docs, but the current implementation feels backwards. I'd expect the values in italics to be the opposite:
* true when `filter[attribute]` is not set
* false when `cfg is not set, if there's anything in filter
* ***true*** when `cfg[attribute]` is a subset of filter
* ***false*** when `filter[attribute]` is a strict subset of cfg

```
Apply a whitelist filter - both filter[attribute] and cfg[attribute] are assumed to be comma-separated
lists, if present. If the filter is present, each item in cfg[filter] must match an item in filter[attribute]
for the function to return true.
```

Reply:
fixed, (i think)